src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Style Guide
+description: Style Guide for ethereum.org
+lang: en
+sidebar: true
+---
+
+# Ethereum.org Style Guide {#style-guide}
+
+Content on [Etheruem.org](http://etheruem.org) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
+
+## Who can submit content to Ethereum.org {#who-can-help}
+
+Anyone! [Ethereum.org](http://ethereum.org) is entirely open source, and many of its best pages are submitted by curious learners who expanded their notes into documentation pages now living on the site!
```

Comment:
Would either remove the redundant linking to the homepage and just make this text, or if we're going to keep the links, they should just link to `/` so it's not treated as an external link.

```suggestion
Anyone! Ethereum.org is entirely open source, and many of its best pages are submitted by curious learners who expanded their notes into documentation pages now living on the site!
```
Same thing with the others below